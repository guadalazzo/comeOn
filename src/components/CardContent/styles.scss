.card__content {
  height: 85%;
  display: flex;
  align-items: center;
  flex-direction: column;
  text-align: center;
  overflow-y: auto;
  h4,p {
    text-align: left;
  }
}