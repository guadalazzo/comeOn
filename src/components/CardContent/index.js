import './styles.scss';
import React from 'react';

const CardContent = ({ children }) => {
    return <section className="card__content">{children}</section>;
};
export default CardContent;
