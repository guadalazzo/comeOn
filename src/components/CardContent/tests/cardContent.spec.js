import React from 'react';
import CardContent from '../index';
import renderer from 'react-test-renderer';


describe('Test CardContent component', () => {
    test('CardContent snapshot', () => {
        const component = renderer.create(<CardContent>Home</CardContent>);
        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
});
