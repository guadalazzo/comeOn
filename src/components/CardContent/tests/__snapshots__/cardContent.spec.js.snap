// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test CardContent component CardContent snapshot 1`] = `
<section
  className="card__content"
>
  Home
</section>
`;
